README.rst

```diff
@@ -49,14 +49,14 @@ To get the git version do
     $ git clone git://github.com/sympy/sympy.git
 
 For other options (tarballs, debs, etc.), see
-http://docs.sympy.org/dev/install.html.
+https://docs.sympy.org/dev/install.html.
```

Comment:
Should this become :ref:`install <install>`
This is regarding issue #15380 in which we should use relative links.
If so I will open a PR.